client/views/app/room.coffee

```diff
@@ -281,18 +281,10 @@ Template.room.helpers
 		# this is to make "bannerData" itself reactive by having it depend directly on the room data.
 		# Then, since that data gets synchronized with the server, the template will be reprocessed
 		# when the data changes.
-		accessPermissions = ChatRoom.findOne(this._id)?.accessPermissions || []
-		Template.instance().updateBannerData(accessPermissions)
-		return Template.instance().bannerData
-
-	# For helpers "classificationId" and "securityBannerText", "this" refers to what is returned
-	# from "bannerData"
-	classificationId: ->
-		return this.get 'classificationId'
-
-	securityBannerText: ->
-		return this.get 'text'
-		
+		roomData = Session.get('roomData' + this._id)
+		Template.instance().accessPermissions.set roomData?.accessPermissions
+		return Template.instance().accessPermissions
```

Comment:
Ok, makes sense. I did try it the other way on my own machine, and it _was_ a little bit confusing because I had to use 'self.data._id' to access the room id, and at least for me, it's never super clear exactly what the data context is if it's not passed explicitly.